pkg/ibmcloudprovider/volume_provider.go

```diff
@@ -78,6 +78,8 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
 			return nil, err
 		}
 		logger.Info("Fetched clusterInfo..")
+	} else {
```

Comment:
can we remove this else condition as csi driver main.go is generic to read the extra label